dataplane/linux/vxlan_mgr.go

```diff
@@ -212,7 +306,7 @@ func (m *vxlanManager) CompleteDeferredWork() error {
 
 // KeepVXLANDeviceInSync is a goroutine that configures the VXLAN tunnel device, then periodically
 // checks that it is still correctly configured.
-func (m *vxlanManager) KeepVXLANDeviceInSync(mtu int) {
+func (m *vxlanManager) KeepVXLANDeviceInSync(mtu, wait int) {
```

Comment:
I'd recommend passing a `time.Duration` here - it's a more obvious API for callers. Then they don't need to look into this function to see how the `wait` is interpreted (is it seconds? minutes? something else?)